gatling-jms/src/main/scala/io/gatling/jms/action/JmsReqReply.scala

```diff
@@ -123,9 +124,9 @@ class JmsReqReplyActor(attributes: JmsAttributes, protocol: JmsProtocol, tracker
       messageProperties.flatMap { props =>
         attributes.message match {
           case BytesJmsMessage(bytes) => bytes(session).map(bytes => client.sendBytesMessage(bytes, props, jmsType))
-          case MapJmsMessage(map)     => map(session).map(map => client.sendMapMessage(map, props, jmsType))
-          case ObjectJmsMessage(o)    => o(session).map(o => client.sendObjectMessage(o, props, jmsType))
-          case TextJmsMessage(txt)    => txt(session).map(txt => client.sendTextMessage(txt, props, jmsType))
+          case MapJmsMessage(map) => map(session).map(map => client.sendMapMessage(map, props, jmsType))
+          case ObjectJmsMessage(o) => o(session).map(o => client.sendObjectMessage(o, props, jmsType))
+          case TextJmsMessage(txt) => txt(session).map(txt => client.sendTextMessage(txt, props, jmsType))
```

Comment:
Those are probably no longer aligned because you formatted with IntelliJ defaults.
Please compile with sbt and code will automatically be formatted.